docs/delivery_methods/ios.md

```diff
@@ -83,13 +83,62 @@ end
 Apple Push Notifications may fail delivery if the user has removed the app from their device. Noticed allows you
 
 ```ruby
-class CommentNotification
+class CommentNotifier < ApplicationNotifier
   deliver_by :ios do |config|
     config.invalid_token = ->(token) { NotificationToken.where(token: token).destroy_all }
   end
 end
 ```
 
+## Updating the iOS app badge
+
+If you're managing the iOS app badge, you can pass it along in the format
+
+```ruby
+class CommentNotifier < ApplicationNotifier
+  deliver_by :ios do |config|
+    config.format = ->(apn) {
+      apn.alert = "Hello world"
+      apn.custom_payload = {url: root_url(host: "example.org")}
+      apn.badge = recipient.notifications.unread.count
+    }
+  end
+end
+```
+
+Another common action is to update the badge after a user reads a notification. This is a great use of the   Noticed::Ephemeral class. Since it's all in-memory, it will perform the job and not touch the databse.
```

Comment:
```suggestion
Another common action is to update the badge after a user reads a notification. 

This is a great use of the Noticed::Ephemeral class. Since it's all in-memory, it will perform the job and not touch the database.
```